tools/tests/radial-menu/test.js

```diff
@@ -148,18 +151,22 @@ describe('Radial Menu Tests', () => {
 			setupRadialMenuTestPage().then(done).catch(err => console.error(err));
 		});
 
-		xit('Should have closed on keyup after opening', function (done) {
+		it('Should have closed on keyup after opening', function (done) {
 			const radialMenu = document.querySelector('gameface-radial-menu');
 
-			dispatchKeyboardEventKeyDown(16, radialMenu);
-
 			// The menu is opened 2 frames after the opening function is called.
-			// More time is apparently needed for it to open and 200 frames seems to not be flaky.
-			waitForStyles(() => {
-				dispatchKeyboardEventKeyUp(16, radialMenu);
-				assert(radialMenu.classList.contains('radial-menu-open') === false, 'Radial menu contains class radial-menu-open.');
-				done();
-			}, 10);
+			dispatchKeyboardEventKeyDown(16, window);
+
+			requestAnimationFrame(() => {
+				requestAnimationFrame(async () => {
```

Comment:
why we need async here? as far as I see there are no awaits inside the function context.

Reply:
Also, changed with `waitForStyles()` exposed from actions.js